pkg/asset/ignition/bootstrap/bootstrap_in_place.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/openshift/installer/pkg/asset"
 	"github.com/openshift/installer/pkg/asset/installconfig"
 	"github.com/openshift/installer/pkg/types"
+
+	"regexp"
```

Comment:
Nit: this should be the 1st import

Reply:
Hi @sadasu, why this import call should be first? any golang best practices or memory optimization I'm not aware of? thks.

Reply:
I don't think this is a golang best practice, but more of a convention we follow in OpenShift. This is the import order: 
- single-name pkg alphabetically ordered
- blank line
- github pkgs alphabetically ordered
- blank line
- k8s pkgs alphabetically ordered
- blank line
- other pkgs in alphabetical order